### ISSUE TITLE
Flux agnostic

### ISSUE THREAD

**Issue by stylesuxx** [weblogixx/react-webpack-template#18]
Hey @weblogixx,
as you know I am right now working on a Redux generator on top of [generator-react-webpack](https://github.com/newtriks/generator-react-webpack), which uses this repo as base.

Since Redux handles things slightly different I came across something:

I think it would be better to move *run.js* out of the components folder and into the *src* folder directly, for the following reasons:
  * I do not think, that this really counts as a component (since it does not extend React.component)
  * Redux has components and containers, containers being smart components, so basically the top component in a Redux context is a container. 
This means that in my Redux generator I have to parse the configs and change the path of the run script, add my run script and the main container, delete the run script from the components dir. So if the *run.js* is moved to the root of *src*, I would only need to rewrite run.js without the need of parsing any files.

**TLDR**: I would like to move **/src/components/run.js** to **/src/index.js** and adapt the config accordingly. If you too think that this makes sense, let me know and I will submit a PR.

**Comment by weblogixx** [comment]
Hi @stylesuxx,

sounds reasonable. I am currently on holidays, but will merge it as soon as I can :+1:

**Comment by weblogixx** [comment]
Hi @stylesuxx,

your pr is merged and available in a couple of hours with version 1.2 of the template.
